### ISSUE TITLE
categories of pretrained weight problem

### ISSUE THREAD

**Issue by jeonjj1** [lyuwenyu/RT-DETR#305]
I am planning to use a pretrained weight file to fine-tune my custom dataset with transfer learning. My dataset has 5 categories, whereas I anticipate that the rt-detr weight file is designed for 80 categories. Therefore, I need to modify the weight file to accommodate only 5 categories, but I am not sure how to open the weight file. I would appreciate your assistance with this.

Additionally, I am curious to know if it is necessary to modify the weight file due to the difference in the number of categories.

**Comment by lyuwenyu** [comment]
You don't need to do this yourself. By default, mismatched weights will be discarded.

https://github.com/lyuwenyu/RT-DETR/blob/main/rtdetr_pytorch/src/solver/solver.py#L142

**Comment by jeonjj1** [comment]
Thank you. However, I am curious if not modifying the pretrained weights will result in longer computation times.

Also, since there are only 5 categories, I wonder if I need to change all instances of num_classes = 80 to 5 in the code, and modify the category items as well. (I want to minimize computation as much as possible.)

**Comment by lyuwenyu** [comment]
> Also, since there are only 5 categories, I wonder if I need to change all instances of num_classes = 80 to 5 in the code, and modify the category items as well. (I want to minimize computation as much as possible.)

Yes, uou need to do so

**Comment by jeonjj1** [comment]
Thank you for your response. 
I am wondering if there is a way to further reduce the model size. I am looking for methods that can significantly improve the speed, even if it means a slight decrease in accuracy.

**Comment by jeonjj1** [comment]
Thank you for your help. I am a master's student studying, so there are many things I don't know. I'm curious if the test set is not used separately because I only see train set and val set in the data, and I wonder what data is used during the test.

**Comment by Caohz678** [comment]
Hello, I’d like to ask where exactly the code for the CCFF module is located. I haven’t been able to find it. Thank you very much!
